DataFormats/TrackerCommon/src/TrackerTopology.cc

```diff
@@ -447,3 +448,5 @@ int TrackerTopology::getITPixelLayerNumber(const DetId &id) const {
   }
   return layer;
 }
+
+TYPELOOKUP_DATA_REG(TrackerTopology);
```

Comment:
This should actually go into `ES_TrackerTopology.cc`, see https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideHowToRegisterESData

Reply:
thank you, I'm taking off reco +1